proto/internal/temporal/server/api/persistence/v1/queues.proto

```diff
@@ -46,8 +46,8 @@ message QueueSliceRange {
     TaskKey exclusive_max = 2;
 }
 
-message ReadQueueMessagesNextPageToken {
-    int64 last_read_message_id = 1;
+message ReadQueueNextPageToken {
+    int64 token = 1;
```

Comment:
Hmm, why rename this to token? It seems like it causes stuttering (e.g. token.token), and it's also not as clear to me.

Reply:
I planned to use this field to store the offset value for ListQueueMessage. In the case of ListQueueMessage, it is not a message ID. Instead of creating a new token message, I thought it is better to reuse this field and rename it to something generic.

Reply:
Yeah, let's make a different proto for this token.

Reply:
Done